include/triton/Tools/LinearLayout.h

```diff
@@ -325,6 +325,7 @@ class LinearLayout {
 
   llvm::MapVector<StringAttr, int32_t /*size*/> outDims;
   bool surjective;
+  bool injective;
```

Comment:
No need for now as all our layouts (not sublayouts, but still) are surjective. Let's implement `isInvertible()`